concent_api/core/admin.py

```diff
@@ -18,22 +18,25 @@ class ActivePassiveDownloadsStateFilter(admin.SimpleListFilter):
 
     def lookups(self, request, model_admin):
         return (
-            ('active', 'Active'),
-            ('passive', 'Passive'),
-            ('active_or_downloads', 'Active or with active downloads')
+            ('active', 'Not timed out active'),
```

Comment:
As discussed - I do not like the new names - they complicate otherwise simple things